backend/src/schema/resolvers/users.spec.js

```diff
@@ -544,3 +551,98 @@ describe('switch user role', () => {
     })
   })
 })
+
+describe('save category settings', () => {
+  beforeAll(async () => {
```

Comment:
May you mind to put this in the `beforeEach` here to get the skipped test running?

Because the db is cleaned `afterEach` test …

Works for me … 😁

Reply:
Thanks for the hint